### PR TITLE
debug: auth workflow

### PR DESCRIPTION
**Pull request by thiskevinwang**
https://app.asana.com/0/1202097197789424/1205523634432684/f

Debug https://github.com/hashicorp/dev-portal/actions/workflows/register-preview-url.yml

### CODE REVIEW

File: scripts/add-preview-url-to-cloud-idp.ts [hunk 1]
```diff
@@ -11,21 +11,26 @@ const fetch = createFetch()
 const vercelApiToken = process.env.VERCEL_API_TOKEN
 const vercelTeamId = process.env.VERCEL_TEAM_ID
 
+// For Vercel API docs
+// see https://vercel.com/docs/rest-api/endpoints#get-a-deployment-by-id-or-url
+
 /** A helper to fetch Vercel Deployment Aliases from Vercel Deploy Preview URLs */
 async function fetchDeploymentAlias(previewUrl: string): Promise<string[]> {
+	console.log('fetchDeploymentAlias', previewUrl)
```
Comment (dstaley) [inline scripts/add-preview-url-to-cloud-idp.ts:19]: 🔪

File: scripts/add-preview-url-to-cloud-idp.ts [hunk 2]
```diff
@@ -11,21 +11,26 @@ const fetch = createFetch()
...
 	)
 
 	url.searchParams.set('teamId', vercelTeamId)
-	url.searchParams.set('withAutomaticAliases', '1')
 
 	const headers = { Authorization: `Bearer ${vercelApiToken}` }
 
 	const res = await fetch(url.toString(), { headers })
 	const data = await res.json()
+	console.log(data)
```
Comment (dstaley) [inline scripts/add-preview-url-to-cloud-idp.ts:33]: 🔪

File: .github/workflows/register-preview-url.yml [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
 name: Register Preview URL with CloudIDP
 
 on:
   deployment_status:
-
+  # workflow_dispatch is here for supporting manual iteration
+  workflow_dispatch:
+    inputs:
+      preview_url:
+        description: 'A Vercel preview url. If not provided, the target_url from the deployment_status event will be used.'
```
Comment (thiskevinwang) [inline .github/workflows/register-preview-url.yml:10]:
```suggestion
        description: 'A Vercel preview url. Only used during manual runs'
```

**Review by dstaley — APPROVED** [review]
approving, just remove the console logs

File: scripts/add-preview-url-to-cloud-idp.ts (hunk 2 above)
Comment (thiskevinwang) [reply scripts/add-preview-url-to-cloud-idp.ts:33]: @dstaley I actually. need this log statement :P

**Pull request merged**